pypot/vrep/io.py

```diff
@@ -317,7 +325,7 @@ def call_remote_api(self, func_name, *args, **kwargs):
 
             err = [bool((err >> i) & 1) for i in range(len(vrep_error))]
 
-            if remote_api.simx_return_novalue_flag not in err:
```

Comment:
@jjehl I'm not sure to understand this modification...

Reply:
Oh, not seen you taught to me, it is just that remote_api.simx_return_novalue_flag = True  . This is exactly the same and I prefer to write less code. 

And for class VrepIOError(Exception):, this is just totaly useless. Bye.